### PR TITLE
Added device 009-129 (air conditioner)

### PR DESCRIPTION
**Pull request by marco-hacs**
I added the device 009-129 which was not present and has new functions (e.g. purifier and smart eye) I also translated it into Italian if it can help... I personally tried all the functions without encountering any problems. Thanks for the wonderful work you did

### CODE REVIEW

File: custom_components/connectlife/translations/en.json
Comment (oyvindwe) [inline custom_components/connectlife/translations/en.json:1]: The new properties must also be added to `strings.json`

**Review by oyvindwe — CHANGES_REQUESTED** [review]
Thank you for your contribution! ❤️ 

Just a small change needed.

**Review by oyvindwe — APPROVED** [review]
I updates strings.json

**Comment by oyvindwe** [comment]
Tip for future PRs: Make a _branch_ in your forks before committing changes, so you can keep `main` in sync with upstream repo.

**Comment by marco-hacs** [comment]
Thanks for the advice on using github. I hadn't given importance to string.json because it still worked correctly. 

Nice to have contributed in my small way

**Pull request merged**